CHANGELOG.md

```diff
@@ -187,6 +187,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - `Trainer.request_dataloader` now takes a `RunningStage` enum instance ([#8858](https://github.com/PyTorchLightning/pytorch-lightning/pull/8858))
 
 
+- `LightningCLI` by default support saving to fsspec filesystem ([#9073](https://github.com/PyTorchLightning/pytorch-lightning/pull/9073))
```

Comment:
what does default mean here?

Reply:
It means that if the default were `multifile: bool = True` and someone has the `default_root_dir` an fsspec then it will give a message that this is not supported. The multifile option is for when as input config files within other config files are given, e.g. a general `config.yaml` which internally references another file say `model.yaml`. If multifile is true, saving the config generates the separate config files.

Reply:
I think this should be in the "Added" section (not "Changed") and be:

```suggestion
- Added `LightningCLI` support for saving multifile configuration files ([#9073](https://github.com/PyTorchLightning/pytorch-lightning/pull/9073))
```

Reply:
Moved it to Added section and changed the text to make it more clear